### ISSUE TITLE
Delegator example - Unable to upload

### ISSUE THREAD

**Issue by boyswan** [paritytech/ink#701]
I'm getting an Extrinsic error when trying to upload the delegator example. The error message in canvas is:

```
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:DispatchError
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:module imports a non-existent function
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:PostInfo:
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:actual_weight=
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:max-weight
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:pays_fee=
2021-02-23 15:22:42.006  DEBUG tokio-runtime-worker runtime:Yes
```

I've also tried with a less complex example, but run into the same error when trying to upload. Using the latest contract build and binaryen.

**Comment by cmichi** [comment]
Thanks for the report, we are looking into it.

Could be related to https://github.com/paritytech/cargo-contract/issues/198.

**Comment by cmichi** [comment]
@boyswan We just released `cargo-contract` 0.9.1, which contains a fix for this. If you run `cargo install cargo-contract --force` everything should be working fine.

**Comment by qaibeibei** [comment]
> @boyswan We just released `cargo-contract` 0.9.1, which contains a fix for this. If you run `cargo install cargo-contract --force` everything should be working fine.

Hello, which version of substrate does ink0.9.1 support?
substrate version 2.0.1

![image](https://user-images.githubusercontent.com/42369700/109093144-d933b400-7752-11eb-97c6-3ad67d9ad65d.png)
![image](https://user-images.githubusercontent.com/42369700/109093184-e8b2fd00-7752-11eb-90a7-23fad100d6ed.png)
